src/components/icon/icon.scss

```diff
@@ -0,0 +1,54 @@
+.bds-icon {
+  display: inline-flex;
+  align-items: center;
+  svg {
+    fill: currentColor;
+    width: 100%;
+    min-width: 100%;
+  }
+}
+
+.bds-icon__size--xxx-large {
+  width: 40px;
+  height: 40px;
+}
+
+.bds-icon__size--xx-large {
+  width: 36px;
```

Comment:
We can also create variables to avoid code duplication, like this:
```scss
$icon-xx-large: 36px;
.bds-icon {
  &__size {
     &--xx-large {
       width: $icon-xx-large;
       height: $icon-xx-large;
     }
}
```

Reply:
resolved